README.md

```diff
@@ -1,2 +1,174 @@
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+<!-- - [❓ FAQ (OPTIONAL)](#faq) -->
+- [📝 License](#license)
+
+
+# 📖 Portfolio Project <a name="about-project"></a>
+
+> A project for my Portfolio.
+
+**Portfolio Project** will showcase my skills and accomplishments as a developer.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The technologies used are HTML/CSS only.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML/CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">N/A</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Key features include:
+
+- **Mobile-First Design**
+- **Responsiveness**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [N/A](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Steps to run: 
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Install this project with: N/A
+
+### Usage
+
+To run the project, execute the following command: N/A
+
+### Run tests
+
+To run tests, run the following command: N/A
+
+### Deployment
+
+You can deploy this project using: N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@abel-tefera](https://github.com/abel-tefera)
+- Twitter: [@twitterhandle](https://twitter.com/abelteferabelay)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/abel-tefera-b1b625153)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project, please star it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
Kindly add the future features section to your readme🙏. The future features section is very important as it helps visitors to your project know what they can potentially collaborate with you on in the future. Make sure to include a few features you will add later on